02-spatial-data.Rmd

```diff
@@ -899,7 +899,7 @@ In this book we focus on **terra**, which replaces the older and (in most cases)
 Before learning about the how **terra**'s class system works, this section describes similarities and differences between **terra** and **stars**; this knowledge will help decide which is most appropriate in different situations.
 
 First, **terra** focuses on the most common raster data model (regular grids), while **stars** also allows storing less popular models (including regular, rotated, sheared, rectilinear, and curvilinear grids).
-While **terra** usually handle one or multi-layered rasters^[It also has an additional class `SpatRasterDataset` for storing many collections of datasets.], the **stars** package provides ways to store raster data cubes -- a raster object with many layers (e.g., bands), for many moments in time (e.g., months), and many attributes (e.g., sensor type A and sensor type B).
+While **terra** usually handles one or multi-layered rasters^[It also has an additional class `SpatRasterDataset` for storing many collections of datasets.], the **stars** package provides ways to store raster data cubes -- a raster object with many layers (e.g., bands), for many moments in time (e.g., months), and many attributes (e.g., sensor type A and sensor type B).
```

Comment:
:+1: 

I would also tentatively suggest further changes, e.g.: 

```
**terra** handles one or multi-layered rasters.^[
It also has an additional class `SpatRasterDataset` for storing many collections of datasets.
]
The **stars** package provides ways to store raster data cubes: a raster object with many layers (e.g., bands), for many moments in time (e.g., months), and many attributes (e.g., sensor type A and sensor type B).
```
The sentence is really long. Any other ideas welcome.